trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'base_decorator'
+
+class TrimmerDecorator < Basedecorator
+  def correct_name
+    @nameable.correct_name[0..9]
```

Comment:
kindly add a condition to the name such that it should have a maximum of 10 characters. If it's longer it should trim the word.

like this 

```ruby
class TrimmerDecorator < Decorator
    # The name has a maximum of 10 characters. If it's longer it should trim the word.
    def correct_name
      @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
    end
end
```